KAS-3754 New user role kanselarij

erikap opened the pull request:
Add new user role kanselarij with same permissions as admin, except for the admin features.

Related PRs:
- [ ] https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/352

erikap committed:
Add permissions for new user role kanselarij

erikap committed:
Sort mock-accounts on role position

sergiofenoll reviewed (approved):
Looks good, failing tests are related to addition of new user, some tests check for number of results in user overview, should be a tiny change.

We may want to change the blocked users from testdata.zip,which were added when the blocking feature was added for testing purposes. Their role is Admin, so now they appear at the top of the mock login page. Maybe we want to set them to Vlaams Parlement role instead, so they appear at the bottom?
![image](https://user-images.githubusercontent.com/22411874/208381756-ff141d30-409e-4415-b598-632c3eb776dd.png)

erikap committed:
Merge branch 'development' into feature/KAS-3754-user-role-kanselarij

erikap committed:
Fix cypress tests

erikap committed:
Add secondary sort by name on mock-login users

erikap committed:
Fix Cypress test on user management

ValenberghsSven commented on cypress/integration/unit/user-organisation-management.spec.js:
```diff
@@ -144,7 +144,7 @@ context('testing user and organisation management', () => {
     it('check filter by group', () => {
-      checkRoleFilterSingle('Admin', 8);
+      checkRoleFilterSingle('Admin', 9);
       checkRoleFilterSingle('Secretarie', 11);
```
I think all of these should be "van 12", since we are filtering out one by one
I'll check locally to avoid multiple jenkins runs

ValenberghsSven commented on app/config/permissions.js:
```diff
@@ -67,6 +68,30 @@ const groups = [
+      'manage-news-items',
+      'manage-decision-publications',
+      'manage-document-publications',
+      'manage-themis-publications',
+      'view-document-version-info',
```
manage settings?

ValenberghsSven committed:
fix cypress test, correct page result count

ValenberghsSven committed:
Merge branch 'feature/KAS-3754-user-role-kanselarij' of github.com:kanselarij-vlaanderen/frontend-kaleidos into feature/KAS-3754-user-role-kanselarij

erikap committed:
Only show orgs settings tab if user has manage-users permission

The permission is already checked in the beforeModel-hook of the
settings.organizations route

ValenberghsSven reviewed (approved):
looks fine, permissions can be finetuned later if needed.

erikap committed:
Remove manage-settings permission from secretarie role

erikap committed:
Merge branch 'development' into feature/KAS-3754-user-role-kanselarij

erikap committed:
Merge branch 'feature/KAS-3754-user-role-kanselarij' of github.com:kanselarij-vlaanderen/frontend-kaleidos into feature/KAS-3754-user-role-kanselarij
